Misc/NEWS.d/next/Tests/2021-12-03-14-19-16.bpo-45695.QKBn2E.rst

```diff
@@ -0,0 +1,3 @@
+The Ubuntu test runner now configures and compiles CPython out of tree. The
```

Comment:
As this will show up in Misc/NEWS for the world, I'd word this as "Our Ubuntu test runner on Github CI now" so that readers have context.

Reply:
I'd either scrap the whole NEWS entry or reduce it to "Out-of-tree builds with a read-only source directory are now tested by CI."

Reply:
Done!